### PR TITLE
[Platform] Update NormalizeDelegations and NormalizeUnbondingDelegations for cosmos and tron, add types package

### PR DESCRIPTION
**Pull request by prazd**

### CODE REVIEW

**Comment by vikmeup** [comment]
This behavior should only happen for `/v2/{coin}/staking/delegations/` but those validators should not be included in `/v2/{coin}/staking/validators` endpoint

File: platform/tron/stake.go [hunk 1]
```diff
@@ -12,6 +12,11 @@ import (
 
 const Annual = 0.74
 
+var unknownStakeValidator = blockatlas.StakeValidator{
+	ID:     "unknown",
```
Comment (vikmeup) [inline platform/tron/stake.go]: just realized - the client need ID as validators address to unstake. Can you specify address to ID?

Comment (prazd) [reply platform/tron/stake.go]: yes, ok

File: platform/cosmos/stake.go [hunk 2]
```diff
@@ -192,3 +192,22 @@ func (p *Platform) Denom() DenomType {
...
+func getUnknownValidator(address string) blockatlas.StakeValidator {
+	return blockatlas.StakeValidator{
+		ID:     address,
+		Status: false,
+		Info: blockatlas.StakeValidatorInfo{
+			Name:        "Decommissioned",
+			Description: "Decommissioned",
+		},
+		Details: blockatlas.StakingDetails{
+			Reward: blockatlas.StakingReward{
```
Comment (vikmeup) [inline platform/cosmos/stake.go:205]: Use `GetDetails()`, but set annual to `0`

Comment (vikmeup) [reply platform/cosmos/stake.go:205]:
```
details := p.GetDetails() 
details.reward.annual = 0
```

File: platform/cosmos/stake.go [hunk 3]
```diff
@@ -192,3 +192,22 @@ func (p *Platform) Denom() DenomType {
...
+				Annual: 0,
+			},
+			LockTime:      1814400,
+			MinimumAmount: "1",
```
Comment (vikmeup) [inline platform/cosmos/stake.go:209]: add:
```
		MinimumAmount: minimumAmount,
		LockTime:      lockTime,
		Type:          blockatlas.DelegationTypeDelegate
```

**Pull request merged**